src/components/portfolio/EditDialog.tsx

```diff
@@ -287,6 +287,21 @@ function reducer(state: State, action: Actions): State {
 }
 
 const useStyles = makeStyles((theme) => ({
+  dialog: {
+    '& .MuiDialog-container': {
+      minWidth: '15rem',
+    },
+    '& .MuiDialog-paper': {
+      maxHeight: '40rem',
+    },
```

Comment:
Is there a good reason to limit the dialog height? If there is a large number of portfolios/stocks, I think editing is user-friendlier if the dialog takes up a larger portion of the screen and less scrolling is necessary. The Material-UI default isn't too bad imo (the max height is 100% minus 32px at the top and 32px at the bottom).

If we decide not to limit the dialog height, we should probably also not limit its width (removing the `maxWidth="lg"` property from the `Dialog` component should do the job).

Reply:
This was a stylistic visual choice, I personally was not a fan of the long window. I can revert this if we think it's the better choice. Removing maxWidth makes no difference as far as I can tell.